tests/testcases/015_check-pods-running.yml

```diff
@@ -28,7 +28,7 @@
     - '(run_pods_log.stdout | from_yaml)["items"] | map(attribute = "status.phase") | unique | list == ["Running"]'
     # Check that all pods are ready
     - '(run_pods_log.stdout | from_yaml)["items"] | map(attribute = "status.containerStatuses") | map("map", attribute = "ready") | map("min") | min'
-    retries: 30
+    retries: 60
```

Comment:
When do you really need 10 mins for pods to be ready? It will just slow down CI

Reply:
Apparently containerd with coreos is slow to download images so it failed alot with containers still in "ContainerCreating" mode...

Reply:
oh that's not good..